news/2017-04-07-weld-243Final.asciidoc

```diff
@@ -0,0 +1,44 @@
+---
+layout: news
+title: Weld 2.4.3.Final
+author: Matej Novotny
+priority: 1
+change_frequency: weekly
+excerpt: Make way for Weld 2.4.3.Final!
+desc: Weld 2.4.3.Final released
+tags: [release]
+---
+:linkattrs:
+
+Even as we get ever closer to CDI 2.0 final version, we shan't forget about our stable 2.4 branch
+Ladies and gentlemen, here comes the next-in-line 2.4 release - Weld 2.4.3.Final!
+
+Notable fixes and improvements:
+
+* Weld now shares proxy classes for built-in of the same type (link:https://issues.jboss.org/browse/WELD-2344[WELD-2344, window="_blank"])
+* Session replcication enhancement - built-in session and conversation scoped beans are now not part of the bean identifier index (link:https://issues.jboss.org/browse/WELD-2343[WELD-2343, window="_blank"])
```

Comment:
"are now not part of the bean identifier index" -> "are not part of the bean identifier index anymore"